deploy/kubernetes/overlays/dev/kustomization.yaml

```diff
@@ -1,5 +1,22 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
 bases:
 - ../../base
 patches:
-- controller_images.yaml
-- node_images.yaml
+- controller_add_snapshotter.yaml
+- controller_always_pull.yaml
+- node_always_pull.yaml
+images:
+- name: REPLACEME/gcp-compute-persistent-disk-csi-driver
+  # Replace this with your private image name
+  newName: gcr.io/gke-release/gcp-compute-persistent-disk-csi-driver
+  newTag: "v0.4.0-gke.0"
```

Comment:
Should this be "latest"?

Reply:
We don't release any official "latest" tags, so I just put our latest supported image.  What I expect developers to do, is to replace both newName and newTag with their own repo and tags.